languageLearning/app/src/main/java/com/example/languageLearning/BridgeActivity.java

```diff
@@ -50,6 +50,9 @@ public void onResponse(JSONObject response) {
                     Intent i = new Intent(BridgeActivity.this, ProfExamActivity.class);
                     i.putExtra("languageChoice", language);
                     startActivity(i);
+                }else{
+                    Intent i = new Intent(BridgeActivity.this, MainMenuActivity.class);
+                    startActivity(i);
```

Comment:
If BridgeActivity is to launch MainMenuActivity, MainMenuActivity shall do a finish() after starting BridgeActivity. Otherwise we get two MainMenuActivity's in the backstack, which can be confusing and frustrating for the user.

Reply:
As the file stands now, we need a finish() call after line 67 in MainMenuActivity.java